test/org/sagebionetworks/bridge/redis/JedisTransactionTest.java

```diff
@@ -0,0 +1,80 @@
+package org.sagebionetworks.bridge.redis;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import java.util.List;
+
+import javax.annotation.Resource;
+
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+@ContextConfiguration("classpath:test-context.xml")
+@RunWith(SpringJUnit4ClassRunner.class)
+public class JedisTransactionTest {
+
+    private final String key1 = JedisTransactionTest.class.getSimpleName() + "Key1";
+    private final String key2 = JedisTransactionTest.class.getSimpleName() + "Key2";
```

Comment:
Do we have shared Redis endpoints? Or does each stack (including each dev's local box) have a separate Redis endpoint? If any of them are shared, we might want to stick some random seed in the key so the tests don't interfere with each other.